aria_data_loaders/aria_data_utils/aria_live_streamer.py

```diff
@@ -524,13 +540,6 @@ def initialize_object_detector(self, outputs: dict = {}, object_labels: list = [
                     belongs to; smaller number means latter the segment time-wise
                 - "object_score_list" - List of Float signifying the detection score
         """
-        self.object_detector = ObjectDetectorWrapper()
```

Comment:
@zephirefaith 

Design Qn:

Here in `initialize_object_detector` we are passing `object_labels+meta_objects` to `self.object_detector._init_object_detector`. This argument within the function is purely getting fed to Owlvit C'tor.

Then we are again explicitly setting `_core_objects` & `_meta_objects` on Line 541 & 542.

Can we just pass 2 separate arguments for object_labels & meta_object_labels to the function `self.object_detector._init_object_detector` instead?

Reply:
ignoring this for now.